### PR TITLE
docs: publish YDB CLI 2.16.0 with release notes

### PR DESCRIPTION
**Pull request by galnat**
### Changelog entry 
Publish YDB CLI 2.16.0 with release notes

### Changelog category 
* Documentation (changelog entry is not required)

### CODE REVIEW

**Review by pnv1 — DISMISSED** [review]

File: ydb/docs/en/core/changelog-cli.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+* Added support for PostgreSQL syntax in `ydb workload tpch` и `ydb workload tpcds` benchmarks
+* Added `-c` option for `ydb workload tpcds run` command to compare the result with expected value and show the diff
+* In the `ydb topic write` command the `--codec` option now has default value RAW
+* Added log events for `ydb tools dump` and `ydb tools restore` commands
+* The `ydb tools restore` command now displays error locations
+
+### Bug fixes
+
+* Fixed progress bar in `ydb workload import` command
+* Fixed a bug where restoring from a backup using --import-data could fail if the partitioning of the table was changed
```
Comment (blinkov) [inline ydb/docs/en/core/changelog-cli.md:23]:
```suggestion
* Fixed the progress bar in the `{{ ydb-cli }} workload import` command.
* Resolved an issue where restoring from a backup using the `--import-data` option could fail if the table's partitioning had changed.
```

File: ydb/docs/ru/core/changelog-cli.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+* Увеличена пропускная способность команды `ydb import file csv` примерно в 3 раза
+* Добавлена поддержка stock нагрузки для колоночных таблиц
+* Добавлена поддержка timestamp’ов в формате ISO-8601 для команд `ydb topiс`
+* В команду `ydb sql` добавлена опция `--explain-ast`, которая выводит AST запроса
+* Добавлена подсветка синтаксиса ANSI SQL в интерактивном режиме `ydb`
+* В команды `ydb workload tpch` и `ydb workload tpcds` добавлена поддержка синтаксиса PostgreSQL
+* В команду `ydb workload tpcds run` добавлена опция `-c` для сравнения результата с ожидаемым значением и отображения различий
+* В команде `ydb topic write` для опции `--codec` теперь по умолчанию установлено значение RAW
+* В команды `ydb tools dump` и `ydb tools restore` добавлено логирование событий
+* В команду `ydb tools restore` добавлен вывод пути до места возникновения ошибки
```
Comment (blinkov) [inline ydb/docs/ru/core/changelog-cli.md:20]:
```suggestion
* Увеличена пропускная способность команды `{{ ydb-cli }} import file csv` примерно в 3 раза.  
* Добавлена поддержка [stock-нагрузки](./reference/ydb-cli/commands/workload/stock.md) для [колоночных таблиц](./concepts/datamodel/table.md#column-oriented-tables).
* Реализована поддержка временных меток в формате [ISO 8601](https://ru.wikipedia.org/wiki/ISO_8601) для команд `ydb topic`.  
* В команду `{{ ydb-cli }} sql` добавлена опция `--explain-ast`, которая выводит AST запроса.  
* Добавлена подсветка синтаксиса ANSI SQL в интерактивном режиме.  
* В команды `{{ ydb-cli }} workload tpch` и `{{ ydb-cli }} workload tpcds` добавлена поддержка синтаксиса PostgreSQL.  
* В команду `{{ ydb-cli }} workload tpcds run` добавлена опция `-c` для сравнения результата с ожидаемым значением и отображения различий.  
* В команды `{{ ydb-cli }} tools dump` и `{{ ydb-cli }} tools restore` добавлено логирование событий.  
* В команду `{{ ydb-cli }} tools restore` добавлено указание места возникновения ошибки.

### Изменения с потерей обратной совместимости

* В команде `{{ ydb-cli }} topic write` для опции `--codec` значение по умолчанию изменено на `RAW`.
```

File: ydb/docs/ru/core/changelog-cli.md [hunk 3]
```diff
@@ -2,6 +2,28 @@
...
+
+### Исправления ошибок
+
+* Исправлен progress bar для команды `ydb workload import`
+* Исправлена ошибка восстановления из резервной копии с использованием `--import-data`, если изменилось партиционирование таблицы.
```
Comment (blinkov) [inline ydb/docs/ru/core/changelog-cli.md:25]:
```suggestion
* Исправлен progress bar для команды `ydb workload import`.
* Устранена ошибка восстановления из резервной копии с использованием опции `--import-data`, возникавшая при изменении партиционирования таблицы.
```

**Review by blinkov — DISMISSED** [review]

**Comment by polRk** [comment]
Missing binary for linux/arm64

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**